app/frontend/schools/coaches/components/school_index/SA_Coaches_CoachEditor.res

```diff
@@ -154,6 +156,14 @@ let make = (~coach, ~closeFormCB, ~updateCoachCB, ~authenticityToken) => {
   let (state, send) = React.useReducerWithMapState(reducer, coach, computeInitialState)
 
   let formId = "coach-create-form"
+
+  // Reload page when admin updates `coach.exited` property
+  let reloadOnExitedChange = if coach->Belt.Option.isSome {
+    Coach.exited(coach->Belt.Option.getExn) != state.exited
+  } else {
+    false
+  }
```

Comment:
`Belt.Option.getExn` is unsafe. 
<img width="820" alt="image" src="https://github.com/pupilfirst/pupilfirst/assets/14979190/f20653e6-75f4-4841-ab8b-df5f4dda700f">

Why not switch over the coach? or use `get_with_default`?